src/components/common/view/WavesBackground.native.js

```diff
@@ -23,5 +23,6 @@ const styles = StyleSheet.create({
     width: '100%',
     height: '100%',
     aspectRatio: getSVGAspectRatio(WavesSVG),
+    transform: [{ rotateY: '180deg' }],
```

Comment:
this is not required , i wrote on the channel that i pushed the fixed svg file

Reply:
@roma-hladilka knows he didn't updated PR yet...